### ISSUE TITLE
Add a new task

### ISSUE THREAD

**Issue by danrabbit** [elementary/tasks#54]
We need a method to add a new task to the selected task list

**Comment by alainm23** [comment]
### My design:

```
------------------------------------------------------------------ -> Gtk.Separator
[ x ] [Task name -> Gtk.Entry]
      [Task note -> Gtk.TextVIew ]
       
      [ datetime buttom ]                         [Cancel]  [Save]
------------------------------------------------------------------ -> Gtk.Separator
[Save button] -> Flat -> text color depending on the color of the list
[Cancel button] -> Flat
```
How should this work?
- Could we add it as a row in the task list?
- At the top or end of the task list as a Gtk.Revealer

**Comment by danrabbit** [comment]
Something I think seemed pretty smooth was the approach of Agenda to just have a text entry at the bottom of the view and you could type something and hit enter and then it creates the task. That kept things predictable and made it easy to add a bunch of tasks at once

**Comment by marbetschar** [comment]
I agree with @danrabbit: I also quite like the approach of having a text entry at the bottom of the list. For now we may simply start with making the task summary editable within the list view (similar to how its done for the list title, without the need of double click).

If done like this, we can simply add additional, but empty row at the end of the list which creates a new task.

**Comment by lainsce** [comment]
I have an idea for this:

Instead of making a new row just for the new task entry, we shift this action to a floating action button in the corner, which when clicked, opens up a `New task…` popover. This button would be colored with the task list's color, and sized 32x32 on LoDPi / 64x64 on HiDPI, for easy discoverability.

Design:
![granite_fab_mockup](https://user-images.githubusercontent.com/4886639/72587739-e423e000-38d4-11ea-9fc5-8d83ebe76d01.png)

**Comment by danrabbit** [comment]
Fixed in #89
